core/templates/dev/head/forms/formBuilder.js

```diff
@@ -26,6 +26,20 @@ oppia.constant(
   'RTE_COMPONENT_SPECS',
   window.GLOBALS.RTE_COMPONENT_SPECS ? window.GLOBALS.RTE_COMPONENT_SPECS : {});
 
+// Service for retrieving width of the image
+oppia.factory('imageComponentWidthService', function() {
```

Comment:
Not really. You can have multiple images with different widths in the RTE once you are done changing their size in the modal, because on clicking done for an image it shows up on the RTE and when you try to add a new image the slider is back to 300.0 (the default value.) Also, once you have images of different widths in the RTE, clicking on them opens the modal and the width shows the right value for the right image. But yeah since it is a global service,  when choosing between already uploaded images using the radio buttons in the modal the widths will show the same value for all images.